src/Toml/Printer.hs

```diff
@@ -81,15 +82,29 @@ prettyKeyValue i = Text.concat . map kvText . HashMap.toList
     showText :: Show a => a -> Text
     showText = Text.pack . show
 
-prettyTables :: Int -> HashMap Key TOML -> Text
-prettyTables i = Text.concat . map prettyTable . HashMap.toList
+prettyTables :: Int -> Text -> PrefixMap TOML -> Text
+prettyTables i pref = Text.concat . map prettyTable . HashMap.elems
```

Comment:
Code in this function looks not very elegant... But I don't see better way to implement it. Need to think more, it's not so obvious...